packages/manager/apps/telecom/src/app/telecom/pack/slots/informations/informations.controller.js

```diff
@@ -52,6 +56,25 @@ export default /* @ngInject */ function PackInformationCtrl(
       );
   }
 
+  function getOrderFollowUp(packName) {
+    const url = `/pack/xdsl/${packName}/orderFollowUp`;
+
+    return $http.get(url).then(({ data: orders }) => {
+      let actualOrder = find(orders, (order) => order.status === 'doing');
+
+      if (!actualOrder) {
+        actualOrder = findLast(orders, (order) => order.status === 'done');
+      }
+
+      if (actualOrder?.doneDate) {
+        actualOrder.doneDateLocale = new Date(
+          actualOrder.doneDate,
+        ).toLocaleString();
+      }
+      return actualOrder;
+    });
+  }
```

Comment:
```suggestion
  function getOrderFollowUp(packName) {
    return $http.get(`/pack/xdsl/${packName}/orderFollowUp`).then(({ data: orders }) => {
      const actualOrder = orders.find(({status}) => status === 'doing') || orders.findLast(({status}) => status === 'done');

      if (actualOrder?.doneDate) {
        actualOrder.doneDateLocale = new Date(
          actualOrder.doneDate,
        ).toLocaleString();
      }
      return actualOrder;
    });
  }
```
You have to use the native fonction in your usage no need lodash.